inventory/byo/hosts.ose.example

```diff
@@ -22,6 +22,7 @@ ansible_ssh_user=root
 # Debug level for all OpenShift components (Defaults to 2)
 debug_level=2
 
+#--------# REVIEW: what are the *current* valid values? #--------#
```

Comment:
I don't think we should be mentioning discontinued types in something that's generally for new installs. So just `origin` and `openshift-enterprise`?